docs/proposals/propagation.md

```diff
@@ -0,0 +1,151 @@
+# HostPath Volume Propagation
+
+## Abstract
+
+A proposal to add support for propagation mode in HostPath volume, which allows
```

Comment:
can you define in this proposal what the modes are and what the implications are? it should be approachable for people not already familiar with kernel internals